configs/components/cpp-pcp-client.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/cpp-pcp-client.git", "ref": "2a0cbb5dbab8c967272b9cd04a0a923b6b42b73f"}
+{"url": "git://github.com/puppetlabs/cpp-pcp-client.git", "ref": "refs/tags/1.1.2"}
```

Comment:
That sha lines up with the tag. Left SHAs for things that have promoted since the release branch (puppet/facter)